### PR TITLE
Add documentation about support of VB.NET

### PR DESCRIPTION
**Pull request by pascalberger**
Add documentation about support of VB.NET. See also #138.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -37,3 +37,7 @@ You can just run `GitVersion.exe` in your repository to see what variables are a
 There are two ways to consume GitVersion, the first is by running GitVersion.exe. The second is an MSBuild task. The MSBuild task is really easy to get up and running, simply install GitVersionTask from NuGet and it will integrate into your project and write out variables to your build server if it's running on one. The exe offers more options and works for not just .net projects.
 
 Read more about [using GitVersion](usage/usage.md)
+
+## Supported language
```
Comment (gep13) [inline docs/index.md]: Should this be supported languages?

File: docs/index.md [hunk 2]
```diff
@@ -37,3 +37,7 @@ You can just run `GitVersion.exe` in your repository to see what variables are a
...
+GitVersion is language and framework independent. 
+The functionality to automatically update the version of projects supports C# projects containing a
```
Comment (gep13) [inline docs/index.md]: This sentence doesn't seem to be finished :smile_cat:

**Comment by pascalberger** [comment]
@gep13 Sorry, commited an outdated version. Is now updated and rebased

**Comment by gep13** [comment]
Perfect :+1:

**Pull request merged**